internal/auth/oidc/repository_auth_method_test.go

```diff
@@ -130,7 +130,7 @@ func Test_upsertAccount(t *testing.T) {
 			idClaims:        map[string]interface{}{"iss": "https://alice.com"},
 			atClaims:        map[string]interface{}{},
 			wantErrMatch:    errors.T(errors.Unknown),
-			wantErrContains: "subject is not present",
+			wantErrContains: "to account subject and it is not present in ID Token",
```

Comment:
Can you add a test which performs the actual mapping and verify things work as expected?

Reply:
ty!